pom.xml

```diff
@@ -218,8 +218,8 @@
     <!-- should follow beam-sdk-java-core version -->
     <joda.time.version>2.10.5</joda.time.version>
 
-    <spark.version>2.4.7</spark.version>
-    <spark-scala.version>2.11</spark-scala.version>
+    <spark.version>3.0.1</spark.version>
```

Comment:
Notice that we could be using a more recent version of Spark here but it is probably a good idea to be aligned with the Spark version that the Data Processing Platform team targets.